e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-company-website-validation.spec.js

```diff
@@ -0,0 +1,98 @@
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const url = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS}`;
+
+      cy.visit(url, {
+        auth: {
+          username: Cypress.config('basicAuthKey'),
+          password: Cypress.config('basicAuthSecret'),
+        },
+      });
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe(`${WEBSITE} error`, () => {
+    it('should display validation errors if company website incorrectly entered', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().type('www');
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 1);
+      partials.errorSummaryListItems().first().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(COMPANY_DETAILS_ERRORS[WEBSITE].INCORRECT_FORMAT);
+        });
+    });
+
+    it('should focus to the company website section when clicking the error', () => {
+      partials.errorSummaryListItemLinks().first().click();
+      companyDetails.companyWebsite().should('have.focus');
+    });
+
+    it('should display the validation error for company website', () => {
+      companyDetails.companyWebsiteError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${COMPANY_DETAILS_ERRORS[WEBSITE].INCORRECT_FORMAT}`);
+        });
+    });
+  });
+
+  describe(`${WEBSITE} no error`, () => {
+    it('should not display validation errors when website is left empty', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear();
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 0);
+    });
+
+    it('should not display validation errors when website is correctly entered', () => {
+      companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+      yesRadioInput().eq(0).click();
+      yesRadioInput().eq(1).click();
+      companyDetails.companyWebsite().clear().type('www.google.com');
+      submitButton().click();
+      partials.errorSummaryListItems().should('have.length', 0);
+    });
+  });
```

Comment:
Could we change this to be more descriptively structured without the `no error` wrapper? e.g:

```js
describe('when website is left empty', () =>   it('should not display validation errors ') );

describe('when website is entered correctly () =>   it('should not display validation errors ') );
```

Obviously indented correcty etc

Reply:
done